README.md

```diff
@@ -107,6 +107,16 @@ tau.forest = causal_forest(X[, selected.vars], Y, W,
                            W.hat = W.hat, Y.hat = Y.hat,
                            tune.parameters = TRUE)
 tau.hat = predict(tau.forest)$predictions
+
+# A simple omnibus test for signal uses tau.hat as a linear predictor for CATE.
```

Comment:
Ideally this update to the `README` would be pulled out into its own PR, since it's unrelated to the OOB prediction change.